src/utils/vaultMappings.ts

```diff
@@ -79,6 +83,18 @@ export const mapVaultApiDataToVault = async (
         }
     );
 
+    const noOtherGroups = groups.filter((g) => g.id !== 'others'); // TODO why is others not resolving?
```

Comment:
I have no idea why the others group never resolves for me. The risk page makes the same call and it also never resolves the page for me...